components/provisioner/go.sum

```diff
@@ -309,6 +309,8 @@ github.com/gardener/gardener v1.11.3/go.mod h1:5DzqfOm+G8UftKu5zUbYJ+9Cnfd4XrvRN
 github.com/gardener/gardener v1.17.1/go.mod h1:uucRHq0xV46xd9MpJJjRswx/Slq3+ipbbJg09FVUtvM=
 github.com/gardener/gardener v1.23.0 h1:fXtV2tKMcSR05GqmxPuhpBhlvUWwBNYfj1ggPRW/kU8=
 github.com/gardener/gardener v1.23.0/go.mod h1:xS/sYyzYsq2W0C79mT98G/qoOTvy/hHTfApHIVF3v2o=
+github.com/gardener/gardener v1.24.0 h1:leLdgrnNBAo/ldJk+mCU1KXnmbHKT1uCarF25U0uf04=
+github.com/gardener/gardener v1.24.0/go.mod h1:Ka66bCr/+tUwkf2Ov4JDMEEvdeeGtHsCfUqw46k7w5A=
```

Comment:
There are multiple versions of Gardener in `go.sum` file. Could you try to run `rm go.sum` and `go mod tidy`? It may clean it a little bit